CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ## Unreleased
 
+### Changed
+- Use Configuration/user.tsconfig (#214)
+
+## 12.0.1 - 2023-09-16
+
+### Fixed
+- Streamline XLIFF files (#209)
```

Comment:
Das gehört nicht zu diesem PR, oder?

Reply:
It was missing, added it as a drive-by. But can move it to a separate PR.